libraries/Ethernet/src/EthernetClient.cpp

```diff
@@ -188,12 +188,12 @@ size_t EthernetClient::write(const uint8_t *buf, size_t size)
 	return size;
 }
 
-int EthernetClient::available()
+int EthernetClient::available(int timeout=0)
 {
```

Comment:
I propose to do this instead,

int EthernetClient::available(int timeout=5000)

for this reason:  Existing programs that uses this function will not be impacted by the timing change.  And for the developers who do not want to have timeout, they can set the parameter to 0 instead of the default of 5000.